cypress/integration/interpretations.cy.js

```diff
@@ -158,9 +158,17 @@ describe('interpretations', () => {
 
         expectInterpretationFormToBeVisible()
 
+        cy.getBySel('interpretations-list')
+            .contains('Update')
+            .should('have.class', 'loading')
+
         cy.getBySel('interpretations-list').contains(
             TEST_INTERPRETATION_TEXT_EDITED
         )
+
+        cy.getBySel('interpretations-list')
+            .contains('Update')
+            .should('not.exist')
```

Comment:
This test was passing on most DHIS2 Core versions, but not on 2.38. I managed to produce it locally and it turned out to be a timing issue. This is a sequential set of tests and the tests _after_ the  `the new interpretation can be edited` were expecting the saved interpretation to be present. However, when these tests started the "Update" button was still loading and the process had not been completed yet. So now I added two steps to verify the side panel goes into a loading state which eventually completes. This should make things more robust.